src/main/webapp/app/shared/shared.module.ts

```diff
@@ -26,6 +27,7 @@ import { ItemCountComponent } from './pagination/item-count.component';
     SortByDirective,
     SortDirective,
     ItemCountComponent,
+    DefineReasonDialogComponent,
```

Comment:
Not sure if this dialog should be placed where it is needed (even if repeated in multiple modules)... just a note

Reply:
It was necessary because it is shared by several modules

Reply:
The same dialog us used on all page when justifying feedback